di_website/templates/includes/heroes/hero-pubs.html

```diff
@@ -44,9 +44,16 @@ <h2 class="hero__subtitle">{{ page.hero_text|content }}</h2>
       {% if page.hero_link and page.hero_link_caption %}
         <a href="{% pageurl page.hero_link %}" class="button button--reverse">{{ page.hero_link_caption }}</a>
       {% endif %}
+
+      {% if page.read_online_button_text %}
```

Comment:
You named this `read_online_text` ... clearly forgot to test it or you'd have noticed the button doesn't show ... make the change in the model as this is the better name.

Reply:
Updated model